app/models/seance.rb

```diff
@@ -1,7 +1,7 @@
 class Seance < ApplicationRecord
   belongs_to :hall
   belongs_to :movie
-  before_validation :set_finishes_at
+  before_validation :start_nil?
```

Comment:
I sugest avoiding calling a method that just calls another method, it's needlesly hard to follow, instead we can for example do it like this:
```suggestion
  before_validation :set_finishes_at, if: :start_nil?
```
And of course we need to modify the `start_nil?` method so that it actually tells us if `start_at` is nill as currently it doesn't do what it says it does. 
A method ending in ? is expected to return true or false

Reply:
I've changed what you pointed out. Thanks :D